contrib/test_jenkins.sh

```diff
@@ -659,6 +659,25 @@ check_make_distcheck() {
 	fi
 }
 
+check_config_h() {
+	echo 1..1 > check_config_h.tap
+
+	srcdir=$PWD/../src
+
+	# Check if all .c files include config.h
+	echo "==== Checking for config.h files in directory $srcdir ===="
+
+	missing=`find $srcdir -name \*.c |xargs grep -Le '#\s*include\s\+"config.h"'`
+	
+	if [ `echo $missing|wc -w` -eq 0 ];
```

Comment:
plz add spaces before and after `|`, like in other places

Reply:
added space